README.md

```diff
@@ -13,21 +13,15 @@ Note: The Docker image [ibmcom/ibm-fhir-schematool](https://hub.docker.com/r/ibm
 More information on installing and running the server is available in the User Guide at https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide.
 
 ### Building on top of the IBM FHIR Server Modules
-Each of the IBM FHIR Server modules are published to a public Maven repository on [JFrog Bintray](https://bintray.com/ibm-watson-health/ibm-fhir-server-releases).
+Each of the IBM FHIR Server modules are published to Maven Central [repo](https://repo1.maven.org/maven2/com/ibm/fhir/)
 
-To use the artifacts from a Maven project:
-1. Add the repository to your pom.xml:
+For Version 4.6.1 and earlier, each of the IBM FHIR Server modules are published as an archive of the repository.
 
-    ```
-    <repositories>
-        <repository>
-            <id>ibm-fhir</id>
-            <url>https://dl.bintray.com/ibm-watson-health/ibm-fhir-server-releases</url>
-        </repository>
-        ...
-    ```
+After Version 4.6.1, you may declare the dependency without a repository as it points to [Maven Central] [https://repo1.maven.org/maven2/com/ibm/fhir]
+
+To use the artifacts from a Maven project:
 
-2. Declare the dependencies:
+1. Declare the dependencies:
```

Comment:
probably no need for the numbered list now since its just one step.  maybe just combine this with the previous line:
> To use the artifacts from a Maven project, declare the dependencies in your pom.xml:

Reply:
simplified in next commit